packages/web-js-sdk/README.md

```diff
@@ -24,6 +24,8 @@ const sdk = descopeSdk({
   sdk.refresh(...), flow.next(...), etc.) in browser storage. In addition, this will
   make `sdk.getSessionToken()` available, see usage bellow bellow */
   persistTokens: true,
+  /* Read Tokens that exist after a succesful authentication, this does not persist incoming tokens and refreshes in browser storage. */
+  readTokens: true,
```

Comment:
`persistTokens` and `readTokens` capabilities overlap, so this api will be tough to explain and reason about

if we are going with this solution, I think that this should be replaced with "enum" string (we can keep `persistTokens` for BC ofc)